Tests/UnitTests/Core/Navigation/PortalNavigationTests.cpp

```diff
@@ -60,12 +60,12 @@ BOOST_AUTO_TEST_CASE(SingleVolumeUpdate) {
   nState.currentVolume = volumeA.get();
   BOOST_CHECK_EQUAL(nState.currentVolume, volumeA.get());
 
-  Acts::Experimental::SingleDetectorVolumeImpl svu(volumeB.get());
+  Acts::Experimental::SingleDetectorVolumeNavigation svu(volumeB.get());
   svu.update(tContext, nState);
 
   BOOST_CHECK_EQUAL(nState.currentVolume, volumeB.get());
 
-  BOOST_CHECK_THROW(Acts::Experimental::SingleDetectorVolumeImpl(nullptr),
+  BOOST_CHECK_THROW(Acts::Experimental::SingleDetectorVolumeNavigation(nullptr),
```

Comment:
I think the exception does not work here, the test is failing

Reply:
Got it.